dockerfile/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM fedora:rawhide
+
+# Prepare environment and install build dependencies
+RUN dnf update -y &&\
+  dnf install -y \
+  'dnf-command(copr)' \
+  'dnf-command(builddep)' \
+  curl \
+  /usr/bin/xargs \
+  rpm-build \
+  python3-pip && \
+  dnf copr enable -y @rhinstaller/Anaconda && \
+  dnf copr enable -y @storage/blivet-daily
+
+RUN curl -L https://raw.githubusercontent.com/rhinstaller/anaconda/master/anaconda.spec.in -o /tmp/anaconda.spec && \
+  sed -i 's/@PACKAGE_VERSION@/0/' /tmp/anaconda.spec && \
+  sed -i 's/@PACKAGE_RELEASE@/0/' /tmp/anaconda.spec && \
+  sed -i 's/%{__python3}/python3/' /tmp/anaconda.spec && \
+  dnf builddep -y /tmp/anaconda.spec && \
+  rpmspec -q --requires /tmp/anaconda.spec | grep -v anaconda | xargs -d '\n' dnf install -y && \
+  dnf clean all
+
+# Install test dependencies
+RUN dnf install -y \
+  e2fsprogs \
+  git \
+  bzip2 \
+  cppcheck \
+  rpm-ostree \
+  pykickstart \
+  python3-pip \
+  python3-lxml \
+  policycoreutils \
+  python3-gobject-base
+
+RUN pip install \
+  pocketlint \
+  coverage \
+  pycodestyle \
+  dogtail \
+  nose-testconfig \
+  rpmfluff
+
+COPY entrypoint.sh /root/entrypoint.sh
+
+VOLUME /workplace
```

Comment:
This isn't being used anywhere, and will just cause pointlessly piling up nameless volumes. Let's please drop this until we actually need it (I don't think we ever will).

Reply:
I thought it's just telling container runtime that here you should expect the anaconda mounted from outside. I guess I missed something here.

Reply:
Yes, for each `VOLUME` that you declare, but don't have a `-v` for, docker/podman will create an "anonymous" volume (`podman volume ls`).